sql/moz-fx-data-shared-prod/telemetry/firefox_android_clients/view.sql

```diff
@@ -0,0 +1,34 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.firefox_android_clients`
```

Comment:
I think these belong in the `fenix` dataset.

Reply:
Makes sense, moved it.
How about renaming this table to `fenix_clients`, would it bring more clarity?